storm-webapp/src/main/java/org/apache/storm/daemon/ui/resources/StormApiResource.java

```diff
@@ -299,6 +299,9 @@ public Response getTopology(@PathParam("id") String id,
                     ),
                     callback
             );
+        } catch (RuntimeException e) {
```

Comment:
why do we need this

Reply:
This provided the callstack that let me debug this.  If a different error occurs, I would like to be able to see the callstack.

Reply:
Okay. 
You might want to update the log message since this function is not only about getting visualization

Reply:
Need to fix this file.  Thanks for catching